mypy/fastparse.py

```diff
@@ -1,4 +1,5 @@
 from functools import wraps
+from inspect import cleandoc
```

Comment:
will do.  I noticed that nearly all functions in these modules were imported using `from x import y` so I was just mimicking the prevailing style, but glad to change it.